pandas/core/frame.py

```diff
@@ -1801,6 +1802,23 @@ def to_stata(self, fname, convert_dates=None, write_index=True,
 
             .. versionadded:: 0.19.0
 
+        version : {114, 117}
+            dta version to use in the output file.  Version 114 can be used
+            read by Stata 10 and later.  Version 117 can be read by Stata 13
```

Comment:
should we just support version 117 and forward? how old are these respective versions?

Reply:
how much simplfication do we get from only supporting new format?